### ISSUE TITLE
PaperUI panel not reflecting external commands

### ISSUE THREAD

**Issue by stevegroom** [raepple/nanoleaf-binding#2]
When I use the paper UI control I can switch on and switch off the aurora panels. 

Cosmetic: 
1) The control panel has a drop down control with off and on states - which correctly control the lights. 
2) The control panel also has a pair of buttons - on and off - again these work correctly, but don't understand why both 1) and 2) are presented. 

Issue: 
3) I have nano leaf aurora light panels that are setup in a HomeKit scene. When the home app (or a configured switch) turns the panel off and on the PaperUI state does not reflect this. 

Updated OpenHabian and the Nanoleaf jar file to the latest build yesterday.

**Comment by raepple** [comment]
I tried to add my item defined for the power channel of the Nanoleaf Controller thing to the Paper UI "Control" UI, but couldn't find out how ...? ;-) (I am only using the iOS mobile app with openHAB)

With my little knowledge for the HABPanel UI, I added a widget for the Power-channel linked item, and tested it by switching the Nanoleaf on and off with the button on the device. After some delay (depending on how long or short you set the refresh rate in the controller configuration settings), the state change was reflected in the widget on the HABPanel UI. Same is true for the iOS app.

Maybe the Paper UI Control UI works different, but I doubt that this is really the case.

PS: The PR got merged today, so we'll see the binding in the next release ;-)
